pkg/auth/auth.go

```diff
@@ -28,6 +28,7 @@ const (
 	ConfluentPlatformPassword    = "CONFLUENT_PLATFORM_PASSWORD"
 	ConfluentPlatformMDSURL      = "CONFLUENT_PLATFORM_MDS_URL"
 	ConfluentPlatformCACertPath  = "CONFLUENT_PLATFORM_CA_CERT_PATH"
+	ConfluentPlatformSSO         = "CONFLUENT_PLATFORM_SSO"
```

Comment:
Let's make sure we add an example of `CONFLUENT_PLATFORM_SSO=true` so users know to pass "true" here instead of something like "1" which is also common